source/lib-intro.tex

```diff
@@ -1614,15 +1614,15 @@
 \begin{codeblock}
 #include <utility>
 
-// Requires: \tcode{std::forward<T>(t)} shall be swappable with \tcode{std::forward<U>(u)}.
+// Mandates: \tcode{std::forward<T>(t)} is swappable with \tcode{std::forward<U>(u)}.
```

Comment:
Both "is swappable with" and "swappable" have semantic requirements, violations of which an implementation cannot be expected to diagnose. To be pedantically correct, these comments both need to Mandate the syntactic requirements and Precondition the semantic requirements. (This admittedly feels like overkill for an example - I have no better idea, though.)

Reply:
How about "requires:"? :-)

Reply:
TFW you remove a term for being too vague and then complain about having to be too precise.

Reply:
Given that getting all those requirements right is critical and that this example concerns a presumably very common customization point, it might be worth doing this right.

@BRevzin: would you mind adding appropriate "precondition" comments?

Reply:
Or we can just use _Preconditions:_, which is how we do this elsewhere in the library spec.